concent_api/core/tests/test_integration_force_payment.py

```diff
@@ -49,7 +49,7 @@ def test_provider_send_force_payment_with_subtask_results_accepted_signed_by_dif
             ),
             self._get_deserialized_subtask_results_accepted(
                 timestamp="2018-02-05 9:00:15",
-                payment_ts="2018-02-05 11:00:16",
+                payment_ts="2018-02-05 8:00:16",
```

Comment:
why changed and why 3 hours? In other place I've seen also 2 hours

Reply:
Answered in https://github.com/golemfactory/concent/pull/1050#discussion_r245933221